config/manager/manager.yaml

```diff
@@ -177,6 +181,9 @@ spec:
       terminationGracePeriodSeconds: 60
       nodeSelector:
         kubernetes.io/os: linux
+      volumes:
+        - name: tmp-volume
+          emptyDir: {}
```

Comment:
Do we need to worry about making sure the data gets wiped once the pod dies?

Reply:
> An emptyDir volume is first created when a Pod is assigned to a node, and exists as long as that Pod is running on that node.... When a Pod is removed from a node for any reason, the data in the emptyDir is deleted permanently.

https://kubernetes.io/docs/concepts/storage/volumes/#emptydir

Reply:
nice!